docker/develop/Dockerfile

```diff
@@ -53,61 +52,93 @@ RUN git clone https://github.com/boostorg/boost /tmp/boost; \
 # install protobuf
 RUN git clone https://github.com/google/protobuf /tmp/protobuf; \
     (cd /tmp/protobuf ; git checkout 80a37e0782d2d702d52234b62dd4b9ec74fd2c95); \
-    cmake -Dprotobuf_BUILD_TESTS=OFF -Dprotobuf_BUILD_SHARED_LIBS=ON -H/tmp/protobuf/cmake -B/tmp/protobuf/.build; \
-    cmake --build /tmp/protobuf/.build --target install -- -j${PARALLELISM}; \
-    ldconfig; \
```

Comment:
That line is removed, why?

Reply:
you need to `ldconfig` only once, in the end: https://github.com/hyperledger/iroha/pull/1064/files#diff-6f13a3dbd7d676a11d65a45353f73feaR226